lib/msf/util/java_deserialization.rb

```diff
@@ -54,20 +54,6 @@ def self.ysoserial_payload(payload_name, command=nil, serial_version_uid: nil)
         bytes[(length_offset-1)..length_offset] = length
       end
 
-      # HACK: Update serialVersionUID if specified
-      if serial_version_uid
-        unless serial_version_uid.is_a?(String)
-          raise ArgumentError, 'serial_version_uid must be a string'
-        end
-
-        if serial_version_uid.length != 8
-          raise ArgumentError, 'serial_version_uid must be 8 bytes'
-        end
-
-        # XXX: This hardcoded value may change!
-        bytes.sub!("\xe3\xa1\x88\xea\x73\x22\xa4\x48", serial_version_uid)
```

Comment:
This won't work if the `CommonsBeanutils1` payload in `data/ysoserial_payloads.json` is updated.